unittests/autoconfig.py

```diff
@@ -56,6 +56,11 @@ class cxx_parsers_cfg(object):
         if 'msvc9' == gccxml.compiler:
             gccxml.define_symbols.append('_HAS_TR1=0')
 
+if cxx_parsers_cfg.gccxml.xml_generator:
```

Comment:
Here's my config file.

```
[gccxml]
xml_generator=castxml
xml_generator_path=/opt/tbs/tbs-CastXML01/root/usr/bin/castxml
compiler=g++
compiler_path=/opt/rh/devtoolset-3/root/usr/bin/g++
```
Without the above change to `autoconfig.py`, I see this output (with DEBUG logging turned on)

```
python test_all.py
Gccxml configured to simulate compiler g++
DEBUG Reading project files: file by file
INFO Parsing source file "core_ns_join_1.hpp" ... 
DEBUG castxml cmd: /opt/tbs/tbs-CastXML01/root/usr/bin/castxml  ...
```
Notice how the second line says `Gccxml` despite castxml being called to do the parsing. With the change, that line becomes
```
Castxml configured to simulate compiler g++
```